src/Symfony/Component/Form/Resources/translations/validators.lb.xlf

```diff
@@ -14,6 +14,126 @@
                 <source>The CSRF token is invalid. Please try to resubmit the form.</source>
                 <target>Den CSRF-Token ass ongëlteg. Versicht wann ech gelift de Formulaire nach eng Kéier ze schécken.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Dëse Wäert ass keng gëlteg HTML5-Faarf.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>W.e.g. e gëltege Gebuertsdatum aginn.</target>
```

Comment:
What does W.e.g stand for?

Reply:
> Awesome. Thank you.
You're welcome

> I've used Google Translate to check these. So of course.. I might be super wrong. I left some comments.
Generally not a good idea to rely on that ... ;-)

> It would be awesome if a native speaker could verify these too.
I asked a colleague of mine, @JohnyGoerend (tagged in the comment above)

I wanted to create a draft PR until he has reviewed the translations, but got a message that those were not allowed. Sorry for the confusion.

Reply:
It's the short form of "wann ech gelift" or "wannechgelift", which means "please".

http://learnluxembourgish.com/blog/2013/11/4/please-wann-ech-gelift

But I just noticed they changed the short form to "wgl." in the new orthography revision (https://www.lod.lu/?WGL1). Thanks for bringing it to my attention.